samply/src/windows/profile_context.rs

```diff
@@ -315,6 +318,9 @@ impl ProfileContext {
             None
         };
         let main_thread_only = profile_creation_props.main_thread_only;
+        let time_range_ns = profile_creation_props
+            .time_range
+            .map(|(start, end)| (start.as_nanos() as u64, end.as_nanos() as u64));
```

Comment:
Could you convert the range ends to profile Timestamps here? Then you don't need to make the `nanos` member public.

Reply:
Oh, yes, good call. Will do. I'll update rust as well, no need to do it